exec-service-wasmer/src/wasmer_opcode_control.rs

```diff
@@ -0,0 +1,203 @@
+use std::{
+    mem,
+    sync::{Arc, Mutex},
+};
+
+use loupe::{MemoryUsage, MemoryUsageTracker};
+use wasmer::{
+    wasmparser::Operator, ExportIndex, FunctionMiddleware, GlobalInit, GlobalType, Instance,
+    LocalFunctionIndex, MiddlewareError, MiddlewareReaderState, ModuleMiddleware, Mutability, Type,
+};
+use wasmer_types::{GlobalIndex, ModuleInfo};
+
+use crate::wasmer_breakpoints::{Breakpoints, BREAKPOINT_VALUE_MEMORY_LIMIT};
+
+const OPCODE_CONTROL_MEMORY_GROW: &str = "opcode_control_memory_grow";
+const OPCODE_CONTROL_MEMORY_GROW_CACHED: &str = "opcode_control_memory_grow_cached";
+
+#[derive(Clone, Debug, MemoryUsage)]
+struct OpcodeControlGlobalIndexes(GlobalIndex, GlobalIndex);
```

Comment:
Optional: make this into a proper struct for readability.

Reply:
Done. Better readability indeed.